Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/APINavPage.xaml.cs

```diff
@@ -0,0 +1,160 @@
+﻿using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.Enums;
+using Amdocs.Ginger.Repository;
+using Ginger.Help;
+using GingerWPF.UserControlsLib.UCTreeView;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace Ginger.BusinessFlowsLibNew.AddActionMenu
+{
+    /// <summary>
+    /// Interaction logic for APINavPage.xaml
+    /// </summary>
+    public partial class APINavPage : Page
+    {
+        Context mContext;
+        ITreeViewItem mItemTypeRootNode;
+
+        public APINavPage(Context context, string itemTypeName, eImageType itemTypeIcon, ITreeViewItem itemTypeRootNode, RoutedEventHandler saveAllHandler = null, RoutedEventHandler addHandler = null, EventHandler treeItemDoubleClickHandler = null)
+        {
+            InitializeComponent();
+
+            mContext = context;
+            mItemTypeRootNode = itemTypeRootNode;
+            GingerHelpProvider.SetHelpString(this, itemTypeName.TrimEnd(new char[] { 's' }));
+
+            xTreeView.TreeTitle = itemTypeName;
+            xTreeView.TreeIcon = itemTypeIcon;
+
+            mContext.PropertyChanged += MContext_PropertyChanged;
+            mContext.Activity.PropertyChanged += Activity_PropertyChanged;
+
```

Comment:
why you track  mContext.Activity.PropertyChanged? i guess all you need is not in Context